tests/test_hooks.py

```diff
@@ -166,3 +167,16 @@ def test_run_hook(self):
 
             hooks.run_hook('post_gen_project', tests_dir, {})
             assert os.path.isfile(os.path.join(tests_dir, 'shell_post.txt'))
+
+    def test_run_failing_hook(self):
+        hook_path = os.path.join(self.hooks_path, 'pre_gen_project.py')
+        tests_dir = os.path.join(self.repo_path, 'input{{hooks}}')
+
+        with open(hook_path, 'w') as f:
```

Comment:
Uhhm, are you changing the test files at runtime? This may break other tests unintentionally... :worried: